contentctl/output/templates/finding_report.j2

```diff
@@ -1,9 +1,9 @@
 
-  | eval devices = [{"hostname": device_hostname, "type_id": 0, "uuid": device.uuid}],
+  | eval devices = [{"hostname": device_hostname, "type_id": 0, "uuid": device.uuid, "ip": device.ip, "mac": device.mac}],
     time = timestamp,
     evidence = {{ detection.tags.evidence_str }},
     message = "{{ detection.name }} has been triggered on " + device_hostname + " by " + {{ actor_user_name }} + ".",
-    users = [{"name": {{ actor_user_name }}, "uuid": actor_user.uuid, "uid": actor_user.uid}],
+    users = [{"name": {{ actor_user_name }}, "uuid": actor_user.uuid, "uid": actor_user.uid, "account_uid": actor_user.account_uid, "email_addr": actor_user.email_addr}],
```

Comment:
Updates requested to finding report

Reply:
This is now nuked

Reply:
Marking as resolved - file was deleted after removal of BA functionality from contentctl